### PR TITLE
Add delete all flag to pipeline command

### PR DESCRIPTION
**Pull request by 16yuki0702**
# Changes
Add delete all flag to the pipeline command feature for https://github.com/tektoncd/cli/issues/233

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
you can specify delete all flag when using pipeline command.
```

### CODE REVIEW

File: docs/man/man1/tkn-clustertask-delete.1 [hunk 1]
```diff
@@ -0,0 +1,66 @@
+.TH "TKN\-CLUSTERTASK\-DELETE" "1" "Aug 2019" "Auto generated by spf13/cobra" ""
```
Comment (16yuki0702) [inline docs/man/man1/tkn-clustertask-delete.1]: This document and ```docs/man/man1/tkn-clustertask.1 ``` is not related this PR, but I got diff when run ```make man``` and ```make docs``` so I included these.

File: pkg/cmd/pipeline/delete.go [hunk 2]
```diff
@@ -27,16 +27,20 @@ import (
...
+		forceDelete: false,
+		deleteAll:   false,
+	}
 	f := cliopts.NewPrintFlags("delete")
 	eg := `
 # Delete a Pipeline of name 'foo' in namespace 'bar'
 tkn pipeline delete foo -n bar
 
-tkn p rm foo -n bar",
+tkn p rm foo -n bar
```
Comment (16yuki0702) [inline pkg/cmd/pipeline/delete.go]: (Memo) I should find and correct these typos after.

File: pkg/cmd/pipeline/delete.go [hunk 3]
```diff
@@ -87,14 +113,19 @@ func checkOptions(opts *deleteOptions, s *cli.Stream, p cli.Params, pName string
 		return nil
 	}
 
-	fmt.Fprintf(s.Out, "Make sure you really want to delete pipeline %q (y/n): ", pName)
+	if opts.deleteAll {
+		fmt.Fprintf(s.Out, "Make sure you really want to delete pipeline and related resources (pipelineruns) %q (y/n): ", pName)
```
Comment (16yuki0702) [inline pkg/cmd/pipeline/delete.go]: Is it better getting all related resources and show these for user before deleting resources??

**Comment by chmouel** [comment]
looks good but at certain point we really need to have some e2e tests,

**Comment by 16yuki0702** [comment]
> looks good but at certain point we really need to have some e2e tests,
yes, I still working on https://github.com/tektoncd/cli/issues/244.
wait please sorry :sweat:

**Comment by chmouel** [comment]
/lgtm

> yes, I still working on #244.

ah i didn't see that one, thanks!

**Comment by 16yuki0702** [comment]
@chmouel @vdemeester Thank you very much !!

**Pull request merged**